dev/openchpl_views.sql

```diff
@@ -148,6 +148,7 @@ FROM openchpl.certified_product a
     ON a.certified_product_id = s.certified_product_id
     LEFT JOIN (SELECT testing_lab_id, name as "testing_lab_name", testing_lab_code from openchpl.testing_lab) q on a.testing_lab_id = q.testing_lab_id
     ;
+	GRANT ALL ON TABLE openchpl.certified_product_details TO openchpl;
```

Comment:
This line isn't required. When running the scripts that uses the files in the `/dev` directory it runs the `grant-all.sql` file, which does the required permission granting